### DISCUSSION TITLE
第6节课作业(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#231] [作业]
请提交作业的链接 Please submit a link to the assignment.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/679416627

**Comment by jayjayhust** [comment]
https://zhuanlan.zhihu.com/p/679619677

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/679842213

**Comment by water2silver** [comment]
https://zhuanlan.zhihu.com/p/680211467
作业与笔记都是这个地址

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/6-OpenCompass-c6a6774a69864e61abecc6b6d538a80f

**Comment by YongXie66** [comment]
课程笔记和课程作业
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson6.pdf

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679959312

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/6
